### DISCUSSION TITLE
"Please check your dataset directories"

### DISCUSSION THREAD

**Discussion by freddydopfel** [d8ahazard/sd_dreambooth_extension#1157] [Q&A] [answered]
Hello!

I'm new to using dreambooth and super excited to get started on my first model. However, after checking all the values, I immediately get this error when I press "train":
Please check your dataset directories.
Duration 00:00:00

I feel like everything should be working fine, and yes, there are a bunch of photos in that directory. Am I doing something wrong?

![image](https://user-images.githubusercontent.com/22718284/229316158-fca61835-0996-472f-986a-be47154454c5.png)

**Comment by wedgeewoo** [comment, marked as answer]
check 
Class Image Generation
Class Images Per Instance Image
its probably at 0

**Reply by vari-ety** [reply]
same issue, Class Images Per Instance Image is set to 5 already

**Reply by freddydopfel** [reply]
Thank you! That worked for me!

vari-ety: did you make sure there aren't any spaces in the directory names? I've heard that causes problems sometimes as well.

**Reply by AbdullahAlfaraj** [reply]
@freddydopfel thank you.
in my case some of the images had spaces in their titles

**Reply by shaochuan** [reply]
In my case, I remove any png files in my folder (or convert them into jpg), and now it starts training.

**Comment by greevak** [comment]
Fixed by completely restarting the UI

**Comment by vari-ety** [comment]
It somehow worked if I select the parent folder and then it worked at the exact folder after I restated sd webui (very weird). But then I ran into this problem. Have no idea now . Output says: 
Exception training model: 'No executable batch size found, reached zero.'.
![image](https://user-images.githubusercontent.com/13483099/229420757-99352642-c420-4bda-b42d-eebc6f6c547b.png)

**Reply by vari-ety** [reply]
fixed by using xformers, seems it was caused by running out of VRAM

**Comment by ablasphemingheretic** [comment]
Resolved by removing capital letters in Instance Token, Instance Prompt, and Sample Image Prompt fields.

**Comment by Proxyseeker2** [comment]
Why is there no solution for this critical problem

**Reply by savekillqqp** [reply]
for me it worked after i restarted the webui-user and i went to the picture folders mother folder for the database directory after removing all spaces in the directory names

**Reply by snailwei** [reply]
OK, finallly found a working solution, the data directory should point to a parent dataset directory, with the model name directory in it!

1. If your model name is `MyModel`, `/path/to/dataset/MyModel` is the real input data (images)
2. Your dataset directory should be `/path/to/dataset/`, trailing slash is needed on Linux

**Comment by sascha1968** [comment]
for me on Mac M1 I had to create in the stable diffusion Folder in the root a training folder and set the directory to Training/cropped instead of /Training/cropped....the / and the fact that the folder is in the root of SD solved it

**Comment by wonghunghing** [comment]
Try rename photos to 1, 2, 3, 4, 5...

**Comment by cbalcunas** [comment]
I changed photos names to remove spaces and make the filename sequential as wonghunghing suggested. Seemed to get a different error in the console. After adding \ to the end of the directory names it is working now.

**Comment by Rickboers** [comment]
don't have your training images name the same as your model name

**Comment by Lecho303** [comment]
help,why my dreambooth tab-input-concept do not have "training wizard(people)/object"?i have pip for sdwebui,thx
